docs/topics/db/sql.txt

```diff
@@ -403,6 +403,31 @@ is equivalent to::
     finally:
         c.close()
 
+Async Connections and cursors
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 5.2
+
+On backends that support async-native connections, you can request an async
```

Comment:
I wonder if we should be specific here and mention that it's only Postgres with ``psycopg`` 3

Reply:
Added in 62510a8163bd42cbf6c5c1c2725d4987da3f42c8